plugins/woocommerce-admin/client/products/product-form-actions.tsx

```diff
@@ -217,7 +226,7 @@ export const ProductFormActions: React.FC = () => {
 								<MenuItem
 									onClick={ onTrash }
 									isDestructive
-									disabled={ ! values.id }
+									disabled={ ! isValidForm || ! values.id }
```

Comment:
I think that this action should be allowed even if the form is not valid. Leaving this line how it was should be enough.

Reply:
Your are right @octaedro, this change is done now. 
But my suggestion is to hide the `Move to trash` action button from the creation page because it does not make sense to show an action to an user that will never be enabled in that page. That action is only needed in the edition page. Does that make sense for you @pmcpinto and @jarekmorawski?

Reply:
It does. 👍 We can hide it for now and bring it back if we hear from the users they need it (which I don't think will happen). We can only show the button once the product is saved as draft or published.